apps/frontend/src/hooks/api/OWNER_DASHBOARD_MIGRATION.md

```
@@ -0,0 +1,439 @@
+const {
+  dashboardStats,
+  propertyStats,
+  tenantStats,
+  leaseStats,
+  activity,
+  isLoading,
+  error
+} = useDashboardPageData()
+```
+
+### NEW: useOwnerDashboardComplete()
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)